packages/pyml_bindgen/pyml_bindgen.0.3.0/opam

```diff
@@ -0,0 +1,56 @@
+opam-version: "2.0"
+synopsis: "Generate pyml bindings from OCaml value specifications"
+maintainer: "Ryan M. Moore"
+authors: "Ryan M. Moore"
+license: ["MIT" "Apache-2.0"]
+homepage: "https://github.com/mooreryan/ocaml_python_bindgen"
+doc: "https://mooreryan.github.io/ocaml_python_bindgen/"
+bug-reports: "https://github.com/mooreryan/ocaml_python_bindgen/issues"
+depends: [
+  "dune" {>= "3.0"}
+  "angstrom" {>= "0.15.0"}
+  "base" {>= "v0.12"}
+  "cmdliner" {>= "1.1.0"}
+  "ppx_let" {>= "v0.12"}
+  "ppx_sexp_conv" {>= "v0.12"}
+  "ppx_string" {>= "v0.12"}
+  "re2" {>= "v0.12"}
+  "stdio" {>= "v0.12"}
+  "ocaml" {>= "4.08.0"}
+  "cmdliner" {= "1.1.0" & with-test}
+  "conf-python-3-dev" {>= "1" & with-test}
+  "core_kernel" {>= "v0.12" & with-test}
+  "ocamlformat" {>= "0.20.0" & < "0.21.0" & with-test}
+  "ppx_assert" {>= "v0.12" & with-test}
+  "ppx_inline_test" {>= "v0.12" & with-test}
+  "ppx_expect" {>= "v0.12" & with-test}
+  "pyml" {with-test}
+  "shexp" {>= "v0.14" & with-test}
+  "bisect_ppx" {dev}
+  "core" {>= "v0.12" & dev}
+  "core_bench" {>= "v0.12" & dev}
```

Comment:
These dependencies don’t seem to be needed anywhere but for bench runs which aren’t in opam.
```suggestion
```

Reply:
I appreciate the review, kit-ty-kate.  I was under the impression that marking these deps as `dev` was how to signal that you only need them if e.g., installing from github or something like that.  As in dependencies used for development (eg bisect_ppx for testing, the other two for benching).  If they are removed, and I try to install it from github in a fresh switch with `opam install . --deps-only --with-doc --with-test`, then the instrumented testing and the benching won't work without installing them by hand.  It's not that big of a deal, as I could put an extra message in the readme about it.   

I did check on bisect_ppx and core_bench and there are quite a few other repositories that have them either as `with-test` or `dev` dependencies (ke and dream are two for example).  It is just a bit unclear to me given the docs and the other examples of packages in opam as to when to include and not include these types of dependencies in `dev` or `with-test` if that is the case.

All that being said, I am fine with removing them if you think it best.